feature/eks-cluster-creation/README.md

```diff
@@ -0,0 +1,48 @@
+## Terraform scripts to provision EKS cluster #
+***
+
+
+The Amazon Elastic Kubernetes Service (EKS) is the AWS service for deploying, managing, and scaling containerized applications with Kubernetes. This repo contain terraform files, to automate the EKS cluster deployment.
+***
+
+#### Requirements:
+- AWS account with the IAM permissions listed on the [EKS module documentation](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/iam-permissions.md)
+- Configured AWS CLI
+- AWS IAM Authenticator
+- kubectl
+- terraform
+
+#### Deployment:
+- Initialize working directory:  
+``` terraform init```
+- Create execution plan:  
+``` terraform plan```
+- Create a cluster:  
+``` terraform apply```
+- Destroy the cluster:  
+``` terraform destroy```
+- Setup kubernetes config for new cluster:
+```aws eks --region $REGION update-kubeconfig --name $CLUSTER_NAME```
```

Comment:
add output related thing here as well

Reply:
Fixed, anyway, kubeconfig is set automatically now via https://github.com/mum0l/epiphany/blob/1442-EKS_cluster_setup/feature/eks-cluster-creation/aws-eks.tf#L35